README.md

```diff
@@ -39,7 +37,7 @@ Start by establishing a connection:
 
 ```python
 import sedonadb
```

Comment:
Can we use sedona.db everywhere? This will help us get all downloads go to apache-sedona[db]?